### PR TITLE
Added new cookbook parse_pdf_docs_for_rag

### PR DESCRIPTION
**Pull request by katia-openai**
Added a new cookbook to show how to parse PDF documents for RAG applications.

This PR includes:
- The new cookbook
- Example data used in the cookbook
- Example files generated while executing the cookbook

### CODE REVIEW

**Review by teomusatoiu — CHANGES_REQUESTED** [review]
Intro: i would make this a bit more verbose. Why is this approach good, pros cons anything else worth noting?

Data Preparation: There's a typo "deducted" instead of "dedicated

Setup: /!\ is not displaying correctly for me. Maybe make the writing bold and write Note: instead?
convert_from_bytes is not used - maybe you can remove?

Processing all documents -4th code cell: I would add a disclaimer that it takes ~2 minutes and what are the different stages that you're executing.

Embedding content:
"Adding data such as the slide and deck title or the doc description at the top of each piece of content so that each independent piece is in context" is a bit unclear to me - not sure what you mean here.

"we rely on separators" - why? what separators? how do we do this?

I would also move all imports to the top. I find it a bit cleaner this way.

ModuleNotFoundError: No module named 'rich' -> add to installs

Wrapping up:
I would add some more practical examples - when would you use this? what are good possible usecases for this Cookbook?

Nit but: You have an extra code cell that is empty at the end - i would remove

File: examples/How_to_combine_GPTv_with_RAG_Outfit_Assistant.ipynb
[the remark this answers is not in the document]
Comment (katia-openai) [reply examples/How_to_combine_GPTv_with_RAG_Outfit_Assistant.ipynb:1]: Yep my bad!

[the remark this answers is not in the document]
Comment (teomusatoiu) [reply examples/How_to_combine_GPTv_with_RAG_Outfit_Assistant.ipynb:1]: It now shows that you've deleted the file. I believe if we merge your PR this file will be deleted on master as well. can you take the master version of this file and put it on your branch as well so there are no "diffs" for it?

File: examples/Parse_PDF_docs_for_RAG.ipynb
Comment (teomusatoiu) [inline examples/Parse_PDF_docs_for_RAG.ipynb:1]: Wrapping up: last sentence is incomplete: 

You can apply the techniques covered in this notebook to multiple use cases, such as assistants that have access to your proprietary data, customer service or FAQ bots that can read from your internal p

**Review by teomusatoiu — CHANGES_REQUESTED** [review]
Looks good. I think generally i would add a bit more text and explanations to make the flow a bit clearer. Otherwise, i think the content is great! 

Happy to approve once you fix the last few pieces: 
- fixing the GPTv with RAG file (it should not show up as part of this PR)
- last sentence in the conclusion is incomplete.

**Pull request closed without merging**